public/beacon/catcher.php

```diff
@@ -1,4 +1,5 @@
 <?php
+
```

Comment:
@korzol 

Could you completely revert this file? I don't think that it's related with the dynamic schema.